Tests/TestCodeGenConfigurations/EmbeddedInTarget-InSchemaModule/Sources/TestApp/AnimalKingdomAPI/Schema/SchemaConfiguration.swift

```diff
@@ -8,7 +8,8 @@
 import ApolloAPI
 
 enum SchemaConfiguration: ApolloAPI.SchemaConfiguration {
-  static func cacheKeyInfo(for type: Object, object: ObjectData) -> CacheKeyInfo? {
-    try? CacheKeyInfo(jsonValue: object["id"])
+  static func cacheKeyInfo(for type: ApolloAPI.Object, object: ApolloAPI.ObjectData) -> CacheKeyInfo? {
+    // Implement this function to configure cache key resolution for your schema types.
+    return nil
```

Comment:
Is this change intended? The objects do support an `id` field.

Reply:
Thanks for catching this. Just reverted it.